### DISCUSSION TITLE
Is it possible to color code all queries?

### DISCUSSION THREAD

**Discussion by cjke** [spatie/ray#197] [Q&A]
It would be neat to be able to do something like:

```
    public function boot()
    {
        ray()->newScreen("Let's roll");
        ray()->showQueries()->blue(); //  the `->blue()` bit 
    }
```

Then, we can filter to "blue" to see all db queries for the current request

**Comment by freekmurze** [comment]
Neat idea, I'll try to add it.

**Reply by cjke** [reply]
Great, cheers. Loving the product so far.
